packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -127,6 +135,9 @@ export class Ec2Service extends BaseService implements IEc2Service, elb.ILoadBal
       maxHealthyPercent: props.daemon && props.maxHealthyPercent === undefined ? 100 : props.maxHealthyPercent,
       minHealthyPercent: props.daemon && props.minHealthyPercent === undefined ? 0 : props.minHealthyPercent ,
       launchType: LaunchType.EC2,
+      propagateTags: props.propagateTaskTagsFrom === undefined ? PropagatedTagSource.SERVICE :
+      (props.propagateTaskTagsFrom === PropagatedTagSource.NONE ? undefined : props.propagateTaskTagsFrom),
```

Comment:
nit: can you indent this out so we can tell it's part of the previous property? Can you also separate each condition and the resulting actions for instance and indent.

Current:
propagateTags: props.propagateTaskTagsFrom === undefined ? PropagatedTagSource.SERVICE :
      (props.propagateTaskTagsFrom === PropagatedTagSource.NONE ? undefined : props.propagateTaskTagsFrom),

Something like:
propagateTags: props.propagateTaskTagsFrom === undefined 
                              ? PropagatedTagSource.SERVICE 
                              : (props.propagateTaskTagsFrom === PropagatedTagSource.NONE 
                                  ? undefined 
                                  : props.propagateTaskTagsFrom),